docs/docs/providers/atlassian.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: Atlassian
+---
+
+# Atlassian API wiki
+
+:::note Working with the Atlassian API?
+Please add your learnings, favorite links and gotchas here by [editing this page](https://github.com/nangohq/nango/tree/master/docs/docs/providers/atlassian.md).
+
+:::
+
+## Using Atlassian with Nango
+
+Provider template name in Nango: `atlassian`  
+Follow our [quickstart](../quickstart.md) to add an OAuth integration with Atlassian in 5 minutes.
+
+## App registration & publishing
+
+**Rating: `Easy & fast`**
+Registering an app takes only a few minutes, and you can start building immediately: [App registration docs](https://developer.atlassian.com/cloud/jira/platform/oauth-2-3lo-apps/)
+
+
+## Useful links
+
+-   [Web API docs (their REST API)](https://developer.atlassian.com/cloud/confluence/oauth-2-3lo-apps)
```

Comment:
Same as for Strava, we should add the link to create a Developer account .

As for the `Web API docs (their REST API)`, let's rename to `OAuth-related docs` if that makes sense.